docs/examples/Example_data_fusion_from_sensors.py

```diff
@@ -0,0 +1,463 @@
+        EKF_tracker.detector = DummyDetector(current=detections)
+        EKF_tracker.__iter__()
+        _, tracks = next(EKF_tracker)
+        ekf_tracks.update(tracks)
+        del tracks
+
+        # Run the Particle Tracker
+        PF_tracker.detector = DummyDetector(current=detections)
+        PF_tracker.__iter__()
+        _, tracks = next(PF_tracker)
+        pf_tracks.update(tracks)
+
+metric_manager.add_data({'KF_tracks': kf_tracks}, overwrite=False)
+metric_manager.add_data({'PF_tracks': pf_tracks}, overwrite=False)
+metric_manager.add_data({'EKF_tracks': ekf_tracks}, overwrite=False)
+
+truths = set(ground_truth_simulator.groundtruth_paths)
+metric_manager.add_data({'truths': truths,
+                         'detections': full_detections}, overwrite=False)
+
+# %%
+# 4) Plot the tracks and the track performances
+# ---------------------------------------------
+# We have obtained the tracks and the ground truths
+# from the various trackers. It is time
+# to visualise the tracks and load the metric manager
+# to evaluate the performances.
+
+plotter = Plotterly()
+plotter.plot_measurements(s1_detections, [0,2], marker= dict(color='blue'),
+                         measurements_label='Radar 1 detections')
+plotter.plot_measurements(s2_detections, [0,2], marker= dict(color='cyan'),
+                         measurements_label='Radar 2 detections')
+plotter.plot_tracks(kf_tracks, [0,2], line= dict(color='grey'), track_label='KF tracks')
+plotter.plot_tracks(ekf_tracks, [0,2], line= dict(color='black'), track_label='EKF tracks')
+plotter.plot_tracks(pf_tracks, [0,2], particle=False, line= dict(color='red'), track_label='PF tracks')
+plotter.plot_ground_truths(truths, [0,2])
+plotter.plot_sensors(sensor1_platform, [0,1], marker=dict(color='black', symbol='129', size=15),
+                     sensor_label='Fixed Platform') # 131 is a diamond cross
+
+plotter.plot_sensors(sensor2_platform, [0, 1], marker=dict(color='darkslategray', symbol='cross',
+                                                           size=15),
+                     sensor_label='Moving Platform')
+plotter.fig
+
+
+# Loaded the plotter for the various metrics.
+from stonesoup.plotter import MetricPlotter
+
+metrics = metric_manager.generate_metrics()
+
+graph = MetricPlotter()
+
+graph.plot_metrics(metrics, generator_names=['OSPA_KF_truths',
+                                             'OSPA_EKF_truths',
+                                             'OSPA_PF_truths'],
+                   color=['green', 'blue', 'orange'])
+graph.axes[0].set(ylabel='OSPA metrics', title='OSPA distances over time')
+graph.fig.show()
+
+
+# %%
+# This concludes this example where we have shown
+# how to perform a measurement fusion using two
+# radars and we have shown the performances
+# of the tracks obtained by an Unscented Kalman filter,
+# an extended Kalman Filter and a particle filter, using
+# distance hypothesiser as data associatior.
+#
```

Comment:
```suggestion
#

```
End of doc line for flake8